docs/proposals/choosing-scheduler.md

```diff
@@ -0,0 +1,165 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.1/docs/proposals/choosing-scheduler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Choosing scheduler in a multi-scheduler configuration
+
+## Introduction
+
+This document proposes a mechanism for assigning a scheduler to a pod
+in a system which is using multiple schedulers as described [here](multiple-schedulers.md).
+In particular, we decribe an admission controller which adds the scheduler name annotation,
+and which takes its configuration from a new API object called `PodPolicy` that determines,
+for each pod, which scheduler is responsible for scheduling that pod, based on various
+characteristics of the pod.
+
+## Implementation
+
+We will create a new admission controller called PodSchedulerPolicy. It will be configured
+using a new API object called `PodPolicy`. We have chosen a generic name and API
+for this object because we want to use the same object for defining other policies based on pod
+characteristics, for example see #17097.
+
+PodSchedulerPolicy will iterate through the `Rules` of the `PodPolicy` and apply the
```

Comment:
PodPolicy isn't a top-level API resource; it's a type that is used in defining the Spec for the admission controller that adds the scheduler name annotation to pods (that admission controller *is* a top-level API resource). So I think you meant what if there is no PodSchedulerPolicy (which is the now-I-realize-it's-terrible name I gave the admission controller). The answer is nothing happens -- no policy is enforced. Should it be otherwise?